packages/carbon-react/.storybook/preview.js

```diff
@@ -8,19 +8,10 @@
 import './styles.scss';
 
 import { configureActions } from '@storybook/addon-actions';
-import * as FeatureFlags from '@carbon/feature-flags';
 import { white, g10, g90, g100 } from '@carbon/themes';
 import React from 'react';
 import { breakpoints } from '@carbon/layout';
-import { settings } from 'carbon-components';
-
-FeatureFlags.merge({
```

Comment:
Out of curiosity, why are the feature flags not needed anymore?

Reply:
Great question! It seems like since our files in `src` were not included in the `sideEffects` field in `package.json` that webpack was incorrectly removing the files that had side effects. Once they were added, any imports we made to components from the project correctly brought in side-effects and now it works as intended 🎉